packages/dialog/index.tsx

```diff
@@ -103,6 +104,8 @@ class Dialog<T extends HTMLElement = HTMLElement> extends React.Component<
     id: 'mdc-dialog',
     open: false,
     role: 'alertdialog',
+    escapeKeyAction: strings.CLOSE_ACTION,
```

Comment:
Can you document these in the Readme, and also document the defaults.

Reply:
Defaults are already written in readme. Do I need to add anything extra?

Reply:
Looks good!